DATA_PREPARATION.md

```diff
@@ -200,15 +200,21 @@ For `key_value_pair` dataset, an additional field `schema` is required to define
 
-| Property    | Type                              | Example                                                                                                                                                        | Required?               |
-| :---------- | :-------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------- |
-| type        | FieldValueType                    | JSON type: string, number, integer, boolean, array, object. Complex object type: bbox, in `ltrb` format, i.e. [left, top, width, height] absolute pixel values. | yes                     |
-| description | string                            | answer to question                                                                                                                                             | no                      |
-| examples    | list[string]                      | examples of field content                                                                                                                                      | no                      |
-| enum        | list[string \| number \| integer] | only work when type is string or number or integer                                                                                                             | no                      |
-| properties  | dict[string, FieldSchema]         | define FieldSchema of each subfield                                                                                                                            | yes when type is object |
-| items       | FieldSchema                       | define the FieldSchema for all items in array                                                                                                                  | yes when type is array  |
-
-More details can be found at [`vision-datasets/vision_datasets/key_value_pair/manifest.py`](vision_datasets/key_value_pair/manifest.py)
+| Property         | Type                      | Example                                                                                                     | Required?               |
+| :--------------- | :------------------------ | :---------------------------------------------------------------------------------------------------------- | :---------------------- |
+| type             | FieldValueType            | JSON type: string, number, integer, boolean, array, object.                                                 | yes                     |
+| description      | string                    | answer to question                                                                                          | no                      |
```

Comment:
Nit: Instead of "answer to question", maybe "describes the field and clarifies/characterizes the field in more detail"?